lib/exabgp/util/__init__.py

```diff
@@ -58,10 +58,16 @@ def padding(n):
 def concat_strs(*items):
 	return ''.join(items)
 
-
 def concat_bytes(*items):
 	return b''.join(items)
 
+# same with iterators/lists
+
+def concat_strs_i(iterable):
```

Comment:
I would rather not have to worry about the data type and have a generic function (as long as the performance are not widely affected) - what is your opinion ?